src/ceph_mon.cc

```diff
@@ -158,25 +158,25 @@ int check_mon_data_empty()
 
 static void usage()
 {
-  cerr << "usage: ceph-mon -i monid [flags]" << std::endl;
-  cerr << "  --debug_mon n\n";
-  cerr << "        debug monitor level (e.g. 10)\n";
-  cerr << "  --mkfs\n";
-  cerr << "        build fresh monitor fs\n";
-  cerr << "  --force-sync\n";
-  cerr << "        force a sync from another mon by wiping local data (BE CAREFUL)\n";
-  cerr << "  --yes-i-really-mean-it\n";
-  cerr << "        mandatory safeguard for --force-sync\n";
-  cerr << "  --compact\n";
-  cerr << "        compact the monitor store\n";
-  cerr << "  --osdmap <filename>\n";
-  cerr << "        only used when --mkfs is provided: load the osdmap from <filename>\n";
-  cerr << "  --inject-monmap <filename>\n";
-  cerr << "        write the <filename> monmap to the local monitor store and exit\n";
-  cerr << "  --extract-monmap <filename>\n";
-  cerr << "        extract the monmap from the local monitor store and exit\n";
-  cerr << "  --mon-data <directory>\n";
-  cerr << "        where the mon store and keyring are located\n";
+  cout << "usage: ceph-mon -i monid [flags]" << std::endl;
```

Comment:
I'm sorry if I wasn't clear. Given we are changing from `cout` to `dout` in most of this file, I thought it was warranted that we'd mimic that for `cerr` to `derr`. But that only makes sense on the portion of code that is potentially useful to have written to the log.

For instance, while it makes sense to output the error condition of `check_mon_data_exists()` (below, c. line 301), it does not make sense, in my opinion, to have argument validation errors outputted to the log. The same goes for `usage`. An additional argument why we should not use `derr` (or `dout`) in those cases, would be that we do not want additional noise on the CLI when the user runs `ceph-mon` on the CLI - `dout` and `derr` will output a lot of information that is only useful in the context of a log, and will be nothing but cumbersome (and possibly unexpected) when the user runs `ceph-mon --help`, for instance.

Does this make sense to you both?